packages/e2e-tests/src/specs/editor/storyDetailsModal/contributorUser.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  createNewStory,
+  uploadMedia,
+  deleteMedia,
+  takeSnapshot,
+  withUser,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Story Details Modal - Contributor User', () => {
+  const openStoryDetailsModal = async () => {
+    await expect(page).toClick('button', { text: 'Submit for review' });
+    await expect(page).toMatchElement('div[aria-label="Story details"]');
+  };
+
+  beforeEach(async () => {
+    await createNewStory();
+    await openStoryDetailsModal();
+  });
+
+  withUser('contributor', 'password');
+
+  it('should allow title update', async () => {
+    await expect(page).toMatchElement('textarea[placeholder="Add title"]');
+
+    await page.type(
+      'textarea[placeholder="Add title"]',
+      'Story Details Modal - contributor'
+    );
+    await page.keyboard.press('Tab');
+
+    // Make sure title is displayed over preview
+    await expect(page).toMatchElement('div[aria-label="Story details"] h3', {
+      text: 'Story Details Modal - contributor',
+    });
+
+    await takeSnapshot(page, 'Story Details Modal - contributor');
+  });
+
+  it('should close modal when story submitted for review', async () => {
+    await expect(page).toClick('div[aria-label="Story details"] button', {
+      text: 'Submit for review',
+    });
+
+    await expect(page).not.toMatchElement('div[aria-label="Story details"]');
+    await expect(page).not.toMatch('Story published.');
+    await expect(page).not.toMatchElement('button', { text: 'Dismiss' });
+  });
+
+  it('should not be able to update Story visibility', async () => {
+    const visibilityDropDownButton = await expect(page).toMatchElement(
+      'button',
+      { text: 'Public' }
+    );
+    await expect(visibilityDropDownButton).toMatch('Public');
+
+    await visibilityDropDownButton.click();
+
+    await expect(page).not.toMatchElement(
+      '[aria-label=" Option List Selector"]'
+    );
+  });
```

Comment:
This is checked [here](https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/e2e-tests/src/specs/editor/contributorUser.js). Do we want it tested in both places?